extensions/amp-subscriptions/0.1/viewer-subscription-platform.js

```diff
@@ -157,7 +167,7 @@ export class ViewerSubscriptionPlatform {
           dataObject: entitlements,
         });
       }
-
+      entitlement.decryptedDocumentKey = entitlements.decryptedDocumentKey;
```

Comment:
1. You've previously made the `decryptedDocumentKey` a const, thus you can't assign it. That's actually a good thing, but you'll likely need to assign it via a constructor.
2. I don't think we can derive the decryption key via JWT - it won't make sense to re-roll the JWT with the decryption key - they are both cryptographic structures of similar strength. It has to be instead provided as a separate response field from the viewer, next to `authorization`.